tests/suites/test_suite_psa_its.function

```diff
@@ -77,7 +83,7 @@ void set_get_remove( int uid_arg, int flags_arg, data_t *data )
     PSA_ASSERT( psa_its_get_info( uid, &info ) );
     TEST_ASSERT( info.size == data->len );
     TEST_ASSERT( info.flags == flags );
-    PSA_ASSERT( psa_its_get( uid, 0, data->len, buffer ) );
+    PSA_ASSERT( psa_its_get( uid, 0, data->len, buffer, &ret_len ) );
```

Comment:
We should probably assert that `ret_len` is equal to `data->len`.

Reply:
Done